content/docs/sidebar.json

```diff
@@ -523,75 +499,49 @@
     "type": "external"
   },
   {
-    "label": "CML",
     "slug": "cml",
-    "source": "cml/index.md",
     "icon": "cml",
-    "children": [
-      {
-        "label": "Get Started (GitHub)",
-        "slug": "start-github"
-      },
-      {
-        "label": "Get Started (GitLab)",
-        "slug": "start-gitlab"
-      },
-      "usage",
-      {
-        "label": "CML with DVC",
-        "slug": "cml-with-dvc"
-      },
-      {
-        "label": "Self-hosted Runners",
-        "slug": "self-hosted-runners"
-      },
-      {
-        "label": "Install as a Package",
-        "slug": "cml-with-npm"
-      }
-    ]
+    "url": "https://cml.dev/doc",
+    "type": "external"
   },
   {
-    "label": "Studio",
     "slug": "studio",
-    "source": "studio/index.md",
     "icon": "studio",
     "children": [
       {
-        "label": "How It Works",
-        "slug": "overview"
+        "slug": "how-it-works",
+        "label": "How It Works"
       },
       "get-started",
       {
-        "label": "User Guide",
         "slug": "user-guide",
-        "source": "user-guide/index.md",
+        "source": false,
         "children": [
           {
-            "label": "Prepare Your Repositories",
-            "slug": "prepare-repositories"
+            "slug": "prepare-repositories",
+            "label": "Prepare Your Repositories"
           },
           {
-            "label": "Views and Experiments",
             "slug": "views",
+            "label": "Views and Experiments",
             "source": false,
             "children": [
               "what-is-a-view",
```

Comment:
This whole sub-section is too buried, especially while the docs are inside dvc.org/doc.

- [ ] Probably we could re-combine some of these pages and put them directly under the Studio UG.